config/logging.py

```diff
@@ -11,7 +11,7 @@
 from enum import Enum
 from pathlib import Path
 from config.config_utils import overwrite_config
-from typing import Any
+from typing import Any, Dict
```

Comment:
nit: no need to use `Dict`. We have python 3.9 so u can just use `dict[str,str]`